cmd/checkcommits/checkcommits.go

```diff
@@ -115,6 +121,17 @@ func checkCommitSubject(config *CommitConfig, commit *Commit) error {
 		return fmt.Errorf("Commit %v: pure whitespace subject", commit.hash)
 	}
 
+	lcSubject := strings.ToLower(subject)
+
+	if strings.HasPrefix(lcSubject, revertPrefix) {
+		// A revert commit is formatted by git(1), not the user,
+		// so always allow it.
+		commit.revertCommit = true
+		commit.subsystem = revertPrefix
+
+		return nil
```

Comment:
Just wanted to confirm that even returning here it is still required the "Fixes" on the commit's body.
I got a little confused by the line below as it also checks for the "Fixes", but apparently on the subject line.

https://github.com/kata-containers/tests/blob/e74a844a2c295a06ef7acf91cc19d3976a5c63ff/cmd/checkcommits/checkcommits.go#L157